src/java-interop/java-interop-logger.cc

```diff
@@ -3,7 +3,7 @@
 
 #include "java-interop-logger.h"
 
-#define DO_LOG(_kind_,_category_,_format_,_args_)			\
+#define LOG_VAARG(_kind_,_category_,_format_,_args_)			\
```

Comment:
If a template isn't good here (it probably isn't), you can use [variadic macros](https://en.cppreference.com/w/cpp/preprocessor/replace) (forms 3 or 4)